documentation/operators/debounce.html

```diff
@@ -315,9 +315,41 @@ <h4>Sample Code</h4>
   {% endlang_operator %}
 
   {% lang_operator RxSwift debounce throttle %}
-     <p>
-      <span style="color:#ff0000">TBD</span>
-     </p>
+     <figure class="variant">
+    <figcaption>
+    <p>
+    RxSwift differences <code>throttle</code> from <code>debounce</code>. While <code>debounce</code> works as defined above, <code>throttle</code> has a different functionality.
+    </p>
+    <p>
+    Returns an Observable that emits the latest item emitted by the source Observable after the specified period of time.
+    </p>
+    <h4>Difference between <code>debounce</code> and <code>throttle</code>:</h4>
+    <ul>
+    	<li>
+    		<code>debounce</code>. Calls the specified function when there hasn't been any events since the last one after the specified period of time:
+    	</li>
+    </ul>
+    <div class="code swift">
+    	<pre>
+    		-----1---2-34-5----6---->
+    		debounce(2)
+    		--------1--------5----6->
+    	</pre>
+    </div>
+    <ul>
+    	<li>
+    		<code>throttle</code>. Calls the specified function when the specified period has passed, regardless of the number of events that have occurred during that time.
```

Comment:
In that case, you should probably move the discussion of the "throttle" operator in RxSwift from this page to http://reactivex.io/documentation/operators/sample.html as it behaves more like sample or throttleLast in other implementations.

Reply:
Hmmm, I think its behaviour is more like `debounce`, indeed, its mention in that (page)[http://reactivex.io/documentation/operators/debounce.html]. Changing its location would make the other `throttle` operators in that (page)[http://reactivex.io/documentation/operators/debounce.html] to be misplaced.

Reply:
There are two throttle operators that are covered on the "sample" page -- throttleFirst and throttleLast. From the description, the RxSwift version of "throttle" looks like it behaves more like one of those than it does the debounce or throttleWithTimeout operators in other languages.

I think RxPHP's "throttle" probably also would be better placed on the "sample" page than on the "debounce" page, based on its description.

Reply:
ok